modules/aws_ecs_ec2/README.md

```diff
@@ -45,12 +46,12 @@ module "retool" {
 To configure the EC instance size, set the `instance_type` input variable (e.g. `t2.large`).
 
 **RDS Instance Class**
-To configure the RDS instance class, set the `instance_class` input variable (e.g. `db.m4.large`).
+To configure the RDS instance class, set the `instance_class` input variable (e.g. `db.m6g.large`).
```

Comment:
Might be too expensive for the default? What's the cost comp

Reply:
This table tells me that m4.large doesn't support PG > v13. I thought this might make it a bit more future proof. https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Types